tests/docker/config/search/search-context.json

```diff
@@ -46,15 +46,6 @@
   "project": {
     "@type": "@id"
   },
-  "valueX": {
```

Comment:
For some reason, putting the type meant that the property was not compacted anymore.
The Json-LD playground has the same behaviour.